asv/__init__.py

```diff
@@ -4,6 +4,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+
```

Comment:
I don't think this is required by flake8, is it?

Reply:
No, it's not, this has been fixed. Thank You, Marc